build-mods-pages.js

```diff
@@ -147,6 +147,20 @@ function filepath(parts) {
 }
 
 for (const mod of mods) {
+    if (mod.versions[0].modJSON.developer == undefined && mod.versions[0].modJSON.developers != undefined) {
+        if (mod.versions[0].modJSON.developers.length == 1) {
+            mod.versions[0].modJSON.developer = mod.versions[0].modJSON.developers[0]
+            mod.versions[0].modJSON.developertwo = mod.versions[0].modJSON.developers[0]
+        } else if (mod.versions[0].modJSON.developers.length == 2) {
+            mod.versions[0].modJSON.developer = mod.versions[0].modJSON.developers[0] + " & " + mod.versions[0].modJSON.developers[1]
+            mod.versions[0].modJSON.developertwo = mod.versions[0].modJSON.developers[0] + " & " + mod.versions[0].modJSON.developers[1]
```

Comment:
theres no reason for this to be different from the case with only 1 developer, both are just joining the array with &

Reply:
in a function